lib/humanize.rb

```diff
@@ -36,6 +36,8 @@ def humanize(options = {})
     end
     if self.class == Float
       decimals = self.to_s.split(/\./, 2).last
+      has_leading_zeros = Array(decimals.match(/^0+/)).first.to_s.length > 0
```

Comment:
I think you could do: `decimals[/0+/]` here and that would return the leading zeroes. For instance:

```ruby
"000042"[/^0+/]
=> "0000"
```

So this code would then be:

```ruby
decimals[/^0+/].length > 0
```

A little shorter :)

Reply:
I think this variable should be called `has_leading_zeroes` too.

Reply:
Haha.. you're right. 😄 I forgot about this shortcut.
Okay, I will edit accordingly and maybe add `to_s` too for when `decimals` doesn't have a leading zeroes.